tests/scripts/thread-cert/command.py

```diff
@@ -66,25 +66,70 @@ def check_address_release(command_msg, destination_node):
     destination_rloc = destination_node.get_ip6_address(config.ADDRESS_TYPE.RLOC)
     assert ipv6.ip_address(destination_rloc) == command_msg.ipv6_packet.ipv6_header.destination_address, "Error: The destination is not RLOC address"
 
-def check_link_request(command_msg):
+def check_tlv_request_tlv(command_msg, check_type, tlv_id):
+    """Verify if TLV Request TLV contains specified TLV ID
+    """
+    tlv_request_tlv = command_msg.get_mle_message_tlv(mle.TlvRequest)
+
+    if check_type == CheckType.CONTAIN:
+        assert tlv_request_tlv is not None, "Error: The msg doesn't contain TLV Request TLV"
+        assert any(tlv_id == tlv for tlv in tlv_request_tlv.tlvs)
+
+    elif check_type == CheckType.NOT_CONTAIN:
+        if tlv_request_tlv is not None:
+            assert any(tlv_id == tlv for tlv in tlv_request_tlv.tlvs) is False
+
+    elif check_type == CheckType.OPTIONAL:
+        if tlv_request_tlv is not None:
+            if any(tlv_id == tlv for tlv in tlv_request_tlv.tlvs):
+                print("TLV Request TLV contains TLV ID: {}".format(tlv_id))
+            else:
+                print("TLV Request TLV doesn't contain TLV ID: {}".format(tlv_id))
+        else:
+            print("TLV Request TLV doesn't contain TLV ID: {}".format(tlv_id))
```

Comment:
The log should be something like above Line 75?

Reply:
I just follow the code in function assertMleMessageContainsOptionalTlv() in message.py. The code in line 75 is used to display an error message. The log here is not error message. When you are running the test case, it will print out lots of these logs. Actually, I think it is better do nothing here.